python/cog/director/director.py

```diff
@@ -298,6 +310,12 @@ def _abort(self, message=None):
         self._should_exit = True
         raise Abort(message)
 
+    def _utilization_thread_loop(self) -> None:
+        while not self._should_exit:
+            with self._tracer.start_as_current_span(name="cog.director.idle") as span:
+                span.set_attribute("idle", self.idle)
```

Comment:
Would this be better as a `threading.Event` rather than a boolean, as we're using it in two different threads?

I know this matches the way we're emitting node utilisation metrics, but I wonder if there's a better way to handle it. This isn't going to give us the data in Honeycomb for a full picture in one graph, because these events are separate to the node utilisation events.

I'll have a noodle about how else we might construct useful data in Honeycomb for this before tomorrow morning.

Reply:
It's ok that this is "racy" because there's only one setter, but I changed it to an event now just so it doesn't confuse anyone.